src/activations.jl

```diff
@@ -58,15 +58,15 @@ const logsigmoid = logσ
 Segment-wise linear approximation of tanh. Cheaper  and  more  computational  efficient version of tanh.
 See [Large Scale Machine Learning](https://ronan.collobert.com/pub/matos/2004_phdthesis_lip6.pdf).
 """
-hardtanh(x) = max(-one(x), min(one(x), x))
+hardtanh(x) = clamp(x, -1, 1)
```

Comment:
Can be bad for some number types

Reply:
You mean `Int32`? We could also decide to delete the tests.

It's 3x faster, because `max` spends time thinking about whether 1 is NaN or not.